### PR TITLE
MongooseIM 4.0.0

### PR DESCRIPTION
**Pull request by NelsonVides**
This PR is the final step for MongooseIM 4.0.0

* Update to CHANGELOG.md fie
* Update to VERSION file
* Update to README files

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,43 @@
+# [MongooseIM 4.0.0](https://github.com/esl/MongooseIM/releases/tag/4.0.0) - 2020-09-30
+## Overview
+MongooseIM 4.0 is all about putting a friendly face to MongooseIM’s amazing features.
+This includes the addition of an Erlang agnostic configuration allowing a broader pool of developers to benefit from MongooseIM.
```
Comment (chrzaszcz) [inline CHANGELOG.md:4]: Erlang-agnostic

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,43 @@
...
+Logging and Kubernetes improvements to be more DevOps friendly and Manager friendly with load testing enabling managers to see the benefits of MongooseIM easily.
```
Comment (chrzaszcz) [inline CHANGELOG.md:5]: DevOps-friendly, Manager-friendly... I even checked some pages about English and it seems that I'm right

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,43 @@
...
+
+## All changes
+
+### Added
+* Agnostic configuration file: MongooseIM can now be configured with an entirely revamped [TOML](https://github.com/toml-lang/toml) configuration file (#2801)
+  * Pesky bugs have been fixed
+  * Implementation details have been abstracted away
+  * Documentation has been improved
+* Structured logging: the main idea of structured logging is that if we have properties of an event, let’s log them as they are, without losing their structure, and format it in a way that is most appropriate for the system we’re sending them to.
+  * Moved away from [lager](https://github.com/erlang-lager/lager), towards [OTP logger](https://erlang.org/doc/man/logger.html) (#2810)
```
Comment (chrzaszcz) [inline CHANGELOG.md:15]: Is the comma necessary?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,43 @@
...
+  * No more logging of man-made strings, we now log structured reports (#2816)
+  * `logfmt` formatter, a log format that's easy to read and write (#2848)
+  * `JSON` formatter, a fully structured format (#2851)
+* [XEP-0215](https://xmpp.org/extensions/xep-0215.html) External Service Discovery (#2870)
+* A rest endpoint for admin users which accepts any stanza, provided it has `from` and `to` attributes (#2858)
```
Comment (chrzaszcz) [inline CHANGELOG.md:20]: A REST endpoint

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,43 @@
+# [MongooseIM 4.0.0](https://github.com/esl/MongooseIM/releases/tag/4.0.0) - 2020-09-30
+## Overview
+MongooseIM 4.0 is all about putting a friendly face to MongooseIM’s amazing features.
+This includes the addition of an Erlang agnostic configuration allowing a broader pool of developers to benefit from MongooseIM.
+Logging and Kubernetes improvements to be more DevOps friendly and Manager friendly with load testing enabling managers to see the benefits of MongooseIM easily.
+
+## All changes
+
+### Added
+* Agnostic configuration file: MongooseIM can now be configured with an entirely revamped [TOML](https://github.com/toml-lang/toml) configuration file (#2801)
```
Comment (chrzaszcz) [inline CHANGELOG.md:10]: For me the word 'agnostic' on it's own is kind of meaningless... that's my impression.

**Review by chrzaszcz — CHANGES_REQUESTED** [review]
Nice desription :-)

You didn't forgot about the `mkdocs.yml`... right?

```
mongoose $ ack '3.7.x'
mkdocs.yml
32:      - '3.7.0 to 3.7.x': 'migrations/3.7.0_3.7.x.md'
```

File: CHANGELOG.md (hunk 3 above)
Comment (NelsonVides) [reply CHANGELOG.md:15]: I think so, those are two different sentences 🤔 @goddammit ?

File: CHANGELOG.md (hunk 5 above)
Comment (NelsonVides) [reply CHANGELOG.md:10]: Maybe "new" would do better?

**Review by chrzaszcz — APPROVED** [review]

**Comment by chrzaszcz** [comment]
I approved, let's wait for CI maybe?

**Review by janciesla8818 — APPROVED** [review]

**Pull request merged**